dagger-reflect/reflect/src/main/java/dagger/reflect/ReflectiveMembersInjector.java

```diff
@@ -75,29 +78,36 @@
 
         methodBindings.put(method, bindings);
       }
+      hierarchyBindings.add(new Bindings(fieldBindings, methodBindings));
```

Comment:
Should we filter empty `Bindings` or let `injectClassMembers` iterate through empty collections?

Reply:
Assuming this was meant to be here and will do:

> Seems worth filtering